sdks/go/examples/xlang/transforms.go

```diff
@@ -17,12 +17,31 @@
 package xlang
 
 import (
+	"reflect"
+
 	"github.com/apache/beam/sdks/go/pkg/beam"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/graph"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/typex"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/util/reflectx"
 )
 
+func init() {
+	beam.RegisterType(reflect.TypeOf((*PrefixPayload)(nil)).Elem())
+}
+
+type PrefixPayload struct {
+	Data string
+}
+
+func Prefix(s beam.Scope, prefix string, addr string, col beam.PCollection) beam.PCollection {
```

Comment:
Consider adding documentation for what the Prefix transform is, and what it's testing/serving as an example of.

We should probably add similar documentation for all the transforms below as well, but that's out of scope for this PR.

Reply:
Done.